go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ovirt/terraform-provider-ovirt
 
-go 1.12
+go 1.15
```

Comment:
Terraform providers should, at this state, force Go version 1.14 because that's the version Terraform is using. They are moving to 1.16 in the future, but that hasn't happened yet.

Reply:
Done